ocaml/src/other/posix_stubs.c

```diff
@@ -62,9 +57,7 @@ void _bs_posix_fadvise(value fd, value offset, value len, value advice) {
                         break;
         }
 
-        enter_blocking_section();
```

Comment:
I don't expect the posix_fadvise to take long enough to have any benifit of releasing/reacquiring the runtime_system, but I could be wrong here...